modules/_canvas-kit-react/package.json

```diff
@@ -50,6 +50,7 @@
     "@workday/canvas-kit-react-form-field": "^3.0.0-alpha.6",
     "@workday/canvas-kit-react-header": "^3.0.0-alpha.6",
     "@workday/canvas-kit-react-icon": "^3.0.0-alpha.6",
+    "@workday/canvas-kit-react-input": "0.0.0",
```

Comment:
This version should match the others.

Reply:
@Parker-Ledoux @MackenzieBerliner-Glasser this has to be `0.0.0` before a release is on npm. This instructs yarn to only look at local files. Changing this will fail CI